radis/lbl/base.py

```diff
@@ -409,6 +409,33 @@ def _add_EvibErot(self, df, calc_Evib_harmonic_anharmonic=False):
         # the radis.db database
         elif self.params.levelsfmt == "radis":
             molecule = self.input.species
+            dbformat = getattr(self.params, 'dbformat', None)
+            if dbformat in ["exomol", "hdf5-radisdb"] and molecule == "OH":
+                # Use ExoMol/OH constants
+                self.profiler.start("fetch_energy_exomol_oh", 2)
+                we = 3737.76  # vibrational constant in cm^-1
+                wexe = 84.88   # anharmonicity constant in cm^-1
+                B = 18.87      # rotational constant in cm^-1
```

Comment:
ok for now but a clean way would be to define a function in the style of `_add_EvibErot_RADIS_cls1`.  
If you don't want to do that for the moment, you should print a Warning and say that this is a temporary implementation